### DISCUSSION TITLE
Add a way to donate inside the app

### DISCUSSION THREAD

**Discussion by yanokwa** [organicmaps/organicmaps#1520] [Ideas]
I would love to donate either one-time or on an ongoing basis in the iOS app. I think having that option in the app would enable more drive by users to pay. I think the payment option should be framed as supporting ongoing project rather than a pure donation. I think the old maps.me app had something similar.

**Comment by biodranik** [comment]
MapsMe had a "disable ads" subscription.

There are several issues with in-app purchases:
1. They are visible to everyone else in the store. Some users may avoid installation by just seeing that there are in-app purchases in the app.
2. Apple and Google take 15% from each donation via in-app purchase.
3. There will be no more visible difference for users between OM and other apps, like OSMand, which sell premium features to users. Expectations of some people may not match all features which our small team can deliver on "donations only" in-app purchases.

We are still investigating the best way to handle it. Maybe apple pay can be added to the website, or something like that.

**Reply by yanokwa** [reply]
I have no data to back this up, but I think it's plausible that the 15% fee might be a good tradeoff if you get a larger volume of traffic.

Anyway, thank you for considering the idea. While you deliberate on next steps, I've gone ahead and sponsored on GitHub. Keep up the great work!

**Reply by biodranik** [reply]
Thank you very much! Was it complex to sponsor via GitHub UI?

**Reply by yanokwa** [reply]
It was easy for me, but I believe it requires a GitHub account, and so will be a barrier for anyone who is not a developer. A [Stripe payment link](https://stripe.com/payments/payment-links) is a lot easier and I believe also supports Apple/Google Pay.
